release-notes/3.13.0.md

```diff
@@ -209,17 +209,17 @@ on [our community Discord](https://rabbitmq.com/discord/).
 
 Release notes are kept under [rabbitmq-server/release-notes](https://github.com/rabbitmq/rabbitmq-server/tree/main/release-notes).
 
-### The Website
+### A New RabbitMQ Website
 
-This release, 3.13.0, introduces a major update to the RabbitMQ website.
-The new site includes:
+This 3.13.0 release includes a change to the RabbitMQ website, a new versioned website.  Access it [here](https://www.rabbitmq.com/).
 
- * Versioning: over time, it will be able to provide documentation on a few most recent release seris, not just the latest one
- * A reworked table of contents and navigation
- * Search
+Some of it's great features include:
 
-The update will go live around the 3.13.0 GA release date.
+* Documentation versions:  When you access the "Docs" tab on the RabbitMQ website, you are viewing documentation for this current 3.13.0 release. The default drop-down option on the top of any page on the far right displays "3.13". You can also access documentation for previous 3.12.x releases. Note, when you select the "3.12" drop-down option, you are taken back to the previous RabbitMQ website, documentation at this site is not versioned, it contains all documentation for previous RabbitMQ releases up to 3.12.x. Over time, the RabbitMQ website will include documentation on a few most recent minor release series but for now as the new website is just launched, it is just documentation for the latest one.
```

Comment:
I'm afraid this is not change log material. We cannot dump a small doc guide in there. It has to be something short, possibly with external links.

Reply:
thanks for your feedback @michaelklishin, I have now shortened it, see commit: https://github.com/rabbitmq/rabbitmq-server/pull/10615/commits/b335930d1882cfa4de35b0229c1dc7fef30259f6
Thanks

Reply:
Actually @michaelklishin instead of my latest commit, can I just link out to the blog: https://682a6c03.rabbitmq-website.pages.dev/blog/2024/01/04/new-website, which is very informative? Is this ok? thanks

Reply:
We can link to https://www.rabbitmq.com/blog/2024/01/04/new-website but not to the development version under `pages.dev`.